website/archiver/utils.py

```diff
@@ -21,6 +22,12 @@
 FILE_HTML_LINK_TEMPLATE = settings.DOMAIN + 'project/{registration_guid}/files/osfstorage/{file_id}'
 FILE_DOWNLOAD_LINK_TEMPLATE = settings.DOMAIN + 'download/{file_id}'
 
+def normalize_unicode_filename(filename):
```

Comment:
Not a blocker, but perhaps this should return a `dict` with keys `nfd` and `nfc` instead of a list?  The later calls to `normalize_unicode_filename(fileobj_metadata['name'])[0]` might be more intuitive if they were written as `normalize_unicode_filename(fileobj_metadata['name'])['nfd'].

Reply:
I attempted refactoring to use a dict, but it caused issues with normalization in certain contexts. Reverting to the list approach for now to ensure functionality, but I will investigate further to understand the cause.